lib/puppetx/simp/iptables.rb

```diff
@@ -317,6 +323,45 @@ def to_s
       #
       def preserve_match(regex = [], components = ['chain', 'jump', 'input_interface', 'output_interface'])
 
+	# A standard list of items that are built into iptables, along with the
+	# standard Docker and Kubernetes rules that should always be preserved
+	always_preserve = Regexp.new('^(' + [
```

Comment:
We might want to detect OS family here and add 'OPENSHIFT' if we are EL

Reply:
OpenShift can run on whatever so that can be included.

Maybe we can break this up into a few chunks.

* `basic` => Usually used built in IPTables items
* `advanced` => More rare IPTables items (NAT, etc...)
* `container` => Container items

All enabled by default.

Unfortunately, anything exposed here also has to be exposed at the top level of the `iptables` module for toggling.

That said, if we can get this verified and through, we can make that a future toggle after discussion.